src/classes/UTIL_CustomSettingsFacade.cls

```diff
@@ -50,6 +52,16 @@ public without sharing class UTIL_CustomSettingsFacade {
     static List<Relationship_Lookup__c> relationshipLookup;
 
     /*******************************************************************************************************
+    * @description Constructor, sets boolean value to determine is User can access Settings page 
+    * 
+    */    
+    public UTIL_CustomSettingsFacade() {
+        User user = [SELECT ProfileId FROM User where Id = :UserInfo.getUserId()]; 
+        Profile profile = [SELECT PermissionsCustomizeApplication FROM Profile Where Id = :user.ProfileId];
```

Comment:
Also, shouldn't this be the pull request in which we create a separate class to be the component's controller and move those methods that don't belong out of UTIL_CustomSettingsFacade?